packages/@uppy/provider-views/src/index.js

```diff
@@ -496,6 +494,25 @@ module.exports = class ProviderView {
     }
   }
 
+  listAllFiles (path, files = null) {
+    files = files || []
+    return new Promise((resolve) => {
+      this.provider.list(path).then((res) => {
```

Comment:
should we add a `catch` for this call. Otherwise what will happen when the call to `this.provider.list` fails?